### PR TITLE
Adds 4 tests

### PR DESCRIPTION
**Pull request by edgararuiz**
Added as many tests I could during my flight, it'll hopefully helps us with our coverage levels.

### CODE REVIEW

**Comment by edgararuiz** [comment]
ptal @hadley

File: tests/testthat/test-sql-query.R [hunk 1]
```diff
@@ -4,3 +4,14 @@ test_that("add_suffixes works if no suffix requested", {
   expect_equal(add_suffixes(c("x", "x"), "y", ""), c("x", "x"))
   expect_equal(add_suffixes(c("x", "y"), "y", ""), c("x", "y"))
 })
+
+test_that("select_query() print method output is as expected", {
+  mf <- select_query(lazy_frame(x = 1, src = simulate_dbi()))
+  expect_equal(
```
Comment (hadley) [inline tests/testthat/test-sql-query.R]: Can you please use `expect_known_output()` here instead?

File: tests/testthat/test-translate-window.R [hunk 2]
```diff
@@ -49,3 +49,19 @@ test_that("can override frame of recycled functions", {
   )
 
 })
+
+test_that("win_rank() is accepted by the sql_translator", {
```
Comment (hadley) [inline tests/testthat/test-translate-window.R]: This just testing the print method?

File: tests/testthat/test-sql-query.R [hunk 3]
```diff
@@ -4,3 +4,14 @@ test_that("add_suffixes works if no suffix requested", {
...
+    paste0(
+      capture.output(print.select_query(mf)),
```
Comment (hadley) [inline tests/testthat/test-sql-query.R]: You should be able to just use `print()` here, I think?

File: tests/testthat/test-translate-window.R (hunk 2 above)
Comment (edgararuiz) [reply tests/testthat/test-translate-window.R]: I think I started with `expect_silent()` but felt like we could do more.  Figured that by testing the print, we could catch unintended changes to that output.

File: tests/testthat/test-sql-query.R (hunk 1 above)
Comment (edgararuiz) [reply tests/testthat/test-sql-query.R]: `expect_known_output()` does not seem to like the test

``` 
library(testthat)
library(dbplyr)

test_that("select_query() print method output is as expected", {
  mf <- select_query(lazy_frame(x = 1, src = simulate_dbi()))
  expect_known_output(
    paste0(
      capture.output(dbplyr:::print.select_query(mf)),
      collapse = " "
    ),
    "<SQL SELECT> From:     df Select:   *"
  )
})
#> Error: Test failed: 'select_query() print method output is as expected'
#> * cannot open the connection
#> 1: expect_known_output(paste0(capture.output(dbplyr:::print.select_query(mf)), collapse = " "), 
#>        "<SQL SELECT> From:     df Select:   *") at <text>:6
#> 2: write_lines(act$out, file)
#> 3: base::writeLines(enc2utf8(text), path, useBytes = TRUE)
#> 4: file(con, "w")
```

<sup>Created on 2019-01-02 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

File: tests/testthat/test-sql-query.R (hunk 3 above)
Comment (edgararuiz) [reply tests/testthat/test-sql-query.R]: It looks like the `print()` method is overridden so the outputs won't match:

Comment (edgararuiz) [reply tests/testthat/test-sql-query.R]: library(testthat)
library(dbplyr)

test_that("select_query() print method output is as expected", {
  mf <- select_query(lazy_frame(x = 1, src = simulate_dbi()))
  expect_equal(
    paste0(
      print(dbplyr:::print.select_query(mf)),
      collapse = " "
    ),
    "<SQL SELECT> From:     df Select:   *"
  )
})
#> <SQL SELECT>
#> From:     df
#> Select:   *
#> NULL
#> Error: Test failed: 'select_query() print method output is as expected'
#> * paste0(print(dbplyr:::print.select_query(mf)), collapse = " ") not equal to "<SQL SELECT> From:     df Select:   *".
#> 1/1 mismatches
#> x[1]: ""
#> y[1]: "<SQL SELECT> From:     df Select:   *"

File: tests/testthat/test-sql-query.R [hunk 4]
```diff
@@ -4,3 +4,8 @@ test_that("add_suffixes works if no suffix requested", {
   expect_equal(add_suffixes(c("x", "x"), "y", ""), c("x", "x"))
   expect_equal(add_suffixes(c("x", "y"), "y", ""), c("x", "y"))
 })
+
+test_that("select_query() print method output is as expected", {
+  mf <- select_query(lazy_frame(x = 1, src = simulate_dbi()))
+  expect_known_output(mf, test_path("test-sql-query-select.txt"), print = TRUE)
```
Comment (hadley) [inline tests/testthat/test-sql-query.R]: I just pushed a change in to this change showing how `expect_known_output()` should work. Can you please adapt the other tests to use the same approach?

File: tests/testthat/test-translate-window.R (hunk 2 above)
Comment (hadley) [reply tests/testthat/test-translate-window.R]: I think this test should live in `test-translate-sql-helpers.r`.

**Pull request merged**